### DISCUSSION TITLE
Join our Discord!

### DISCUSSION THREAD

**Discussion by hlhr202** [Atome-FE/llama-node#59] [Announcements]
Hi, we just opened a discord server for discussing various topics about llama-node and above!
Also, I am warmly welcome if anyone can join us and contribute to this project.

Thanks for your attention!

[Discord invitation link](https://discord.gg/dKFeCwfsDk)

**Comment by DavidBDiligence** [comment]
<img width="549" alt="Screenshot 2023-05-25 at 12 55 52 PM" src="https://github.com/Atome-FE/llama-node/assets/106702396/c016cfac-060d-44cd-a0b9-d03f0e6ca3ea">
